gql/filter.go

```diff
@@ -162,7 +172,7 @@ func newListFilter(arg interface{}) (*listFilter, error) {
 		return nil, err
 	}
 
-	return &listFilter{fieldName: lf.Field, op: op}, nil
+	return &listFilter{fieldName: lf.Field, op: op, json: lf}, nil
```

Comment:
here instead of putting in the `json: lf` you would have `operationName: lf.Operation, argument: lf.Argument`

Though there is a bunch going on today so maybe that doesn't work quite as I envision it?